### PR TITLE
docs: Minor textual improvements in translation_strings_policy.md

### PR DESCRIPTION
**Pull request by merland**
Found a few places where the reading flow was interrupted by minor grammar and punctuation issues.

### CODE REVIEW

File: doc/translation_strings_policy.md [hunk 1]
```diff
@@ -33,25 +33,25 @@ General recommendations
...
 
-Try to write translation strings in an understandable way, for both the user and the translator. Avoid overly technical or detailed messages
+Try to write translation strings in an understandable way, for both the user and the translator. Avoid overly technical or detailed messages.
 
 ### Do not translate internal errors
 
-Do not translate internal errors, or log messages, or messages that appear on the RPC interface. If an error is to be shown to the user,
-use a translatable generic message, then log the detailed message to the log. E.g. "A fatal internal error occurred, see debug.log for details".
+Do not translate internal errors, log messages, or messages that appear on the RPC interface. If an error is to be shown to the user,
+use a translatable generic message, then log the detailed message to the log. E.g "A fatal internal error occurred, see debug.log for details".
```
Comment (hebasto) [inline doc/translation_strings_policy.md]: "E.g." should be followed by a comma (as we use American English for docs).
Ref: https://en.wikipedia.org/wiki/List_of_Latin_phrases_(E)#e.g.

File: doc/translation_strings_policy.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ This adds `<numerusform>`s to the respective `.ts` file, which can be translated
         </translation>
     </message>
 
-Where it is possible try to avoid embedding numbers into the flow of the string at all. e.g.
+Where possible, try to avoid embedding numbers into the flow of the string at all. e.g.
```
Comment (hebasto) [inline doc/translation_strings_policy.md]: Same here.
Also the first letter should capitalized after a full stop, right?

**Review by hebasto — CHANGES_REQUESTED** [review]

File: doc/translation_strings_policy.md (hunk 1 above)
Comment (merland) [reply doc/translation_strings_policy.md]: good catch, will update!

File: doc/translation_strings_policy.md (hunk 2 above)
Comment (merland) [reply doc/translation_strings_policy.md]: good catch, will update!

**Comment by fanquake** [comment]
Please squash your changes, and remove any @ mentions from commit messages.

**Comment by merland** [comment]
Squashed and fixed commit message.

**Comment by fanquake** [comment]
utACK a94e470

Please keep your commit messages more specific in future. i.e `docs: textual improvements in translation_strings_policy.md`.

**Pull request merged**